index.js

```diff
@@ -43,8 +43,11 @@ module.exports = {
         route = resolveRouteImport(`${globOptions.cwd}/${file}`);
         route = _.defaultsDeep(route, options.defaultRoute, defaultRoute);
 
-        if (options.pathPrefix) {
-          route.path = options.pathPrefix + route.path;
+        if (options.prefix) {
+          if (route.path.charAt(0) !== '/') {
```

Comment:
I like `path.posix.join` from https://stackoverflow.com/questions/16301503/can-i-use-requirepath-join-to-safely-concatenate-urls